static/styles/portico/markdown.css

```diff
@@ -62,6 +61,7 @@
     h3 {
         font-weight: 700;
         user-select: none;
+        scroll-margin-top: 70px;
```

Comment:
This adds a scroll margin for the element so that when the links are opened using hash, it correctly positions the page.